power_grid_model_c/power_grid_model/include/power_grid_model/common/three_phase_tensor.hpp

```diff
@@ -92,6 +121,7 @@ template <symmetry_tag sym>
 using RealTensor = std::conditional_t<is_symmetric_v<sym>, double, three_phase_tensor::Tensor<double>>;
 template <symmetry_tag sym>
 using ComplexTensor = std::conditional_t<is_symmetric_v<sym>, DoubleComplex, three_phase_tensor::Tensor<DoubleComplex>>;
+using ComplexTensor4 = three_phase_tensor::Tensor4<DoubleComplex>;
```

Comment:
Missed a thing: Only real part of the ComplexTensor4 gets used everywhere. So renaming this (and all places to RealTensor4 might do the job too)

Reply:
Okay, my bad. I do see its being used in the complex form. This can be resolved without changing anything.